modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -0,0 +1,354 @@
+[cols="1,1", options="header"]
+.SLE Micro Products - WebUI
+|===
+
+| OS Version
+| Product Name
+
+| {slemicro} 5.0 {x86_64}
+| SUSE Linux Enterprise Micro 5.0 aarch64
+
+| {slemicro} 5.0 {arm64}
+| SUSE Linux Enterprise Micro 5.0 x86_64
+
+|===
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[sle-channels-cli]]
+[cols="1,1,1", options="header"]
+.SLE Micro Products - CLI
+|===
+
+| OS Version
+| Base Channel
+| Updates Channel
+
+| {slemicro} 5.0 {x86_64}
+| suse-microos-5.0-pool-x86_64
+| suse-microos-5.0-updates-x86_64
+
+| {slemicro} 5.0 {arm64}
+| suse-microos-5.0-pool-aarch64
+| suse-microos-5.0-updates-aarch64
+
+|===
+
+include::snippets/addchannels_vendor_cli.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+[[microos-channels-cli]]
+[cols="1,1,1,1", options="header"]
+.MicroOS Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Updates Channel
+
+| {opensuse}{nbsp}{microos}
+| opensuse_tumbleweed
+| opensuse_tumbleweed-non-oss
+| opensuse_tumbleweed-update
+
+|===
+
+include::snippets/addchannels_novendor_cli.adoc[]
+
+endif::[]
+
+
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+endif::[]
+
+include::snippets/check_sync_cli.adoc[]
+
+[IMPORTANT]
+====
+{slemicro} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+
+ifeval::[{uyuni-content} == true]
+
+== Trust Certificates Keys on Clients
+
+{opensuse}{nbsp}{microos} is not yet fully enabled, so there are some manual steps to trust the {productname} SSL certificate on {microos} clients.
+
+
+// Not tested. From https://etherpad.nue.suse.com/p/uyuni-microos. --LKB 2021-04-15
+
+
+.Procedure: Manually Trusting SSL Certificates
+. On the client, at the command prompt, as root, retrieve the SSL certificate file from the server:
++
+----
+curl -k https://uyuni-server.hispa-net.com/pub/RHN-ORG-TRUSTED-SSL-CERT -o /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT
+----
+. Update the certificates on the client:
++
+----
+update-ca-certificates
+----
+. Install the required packages:
++
+----
+transactional-update pkg install salt-minion dmidecode
+----
+. Reboot the client.
+  If a message is shown indicating that there is a conflict with ``busybox-hostname``, click btn:[Deinstallation of busybox-hostname].
+
+
+You also need to configure Salt on the client to connect correctly to the {productname} Server.
+
+
+
+.Procedure: Configure Salt for MicroOS Clients
+. Create a new file called [path]``/etc/salt/minion.d/susemanager-transactional.conf`` with this content:
++
+----
+module_executors:
+- transactional_update
+- direct_call
+----
+. Update Salt to the correct version for the {opensuse} {microos} repository.
```

Comment:
This should be Pau's repo (for SLE Micro) and not mine (for openSUSE Micro, which is in fact not needed at all after some tests I did).

Pau's repo should not even be needed because the bug that a affects salt should be fixed this week or the next week.

Reply:
what's the URL?

Reply:
No need to mention that, a fixed Salt package should be there soon.

What I would do is mentioning "if Salt is failing to install any software, then you are using an old version of Salt"